pom.xml

```diff
@@ -255,10 +263,12 @@
                               <goal>verify</goal>
                             </goals>
                             <configuration>
+                            <forkCount>1</forkCount>
+        					<reuseForks>false</reuseForks>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Tab characters found (this is the first occurrence) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=xml%3AIllegalTabCheck)